packages/react-scripts/template/README.md

```diff
@@ -2565,7 +2565,7 @@ If this doesn’t happen, try one of the following workarounds:
 
 If none of these solutions help please leave a comment [in this thread](https://github.com/facebook/create-react-app/issues/659).
 
-### `npm test` hangs on macOS Sierra
+### `npm test` hangs or crashes on macOS Sierra
```

Comment:
Also need to fix the link in table of contents above.

Reply:
Yeah noticed just after I pushed. Fixed.